cli/internal/fs/turbo_json.go

```diff
@@ -465,6 +472,12 @@ func (btd *BookkeepingTaskDefinition) UnmarshalJSON(data []byte) error {
 		// Note that we don't require Inputs to be sorted, we're going to
 		// hash the resulting files and sort that instead
 		btd.definedFields.Add("Inputs")
+		// TODO: during rust port, this should be moved to a post-parse validation step
+		for _, input := range task.Inputs {
+			if filepath.IsAbs(input) {
+				log.Printf("[WARNING] Using an absolute path in \"inputs\" (%v) will not work and will be an error in a future version", input)
+			}
```

Comment:
I really wish we had the infrastructure to make this an error _now_, and add a config to opt into the old (incorrect) behavior.